bigtable/admin.go

```diff
@@ -119,6 +120,74 @@ func (ac *AdminClient) backupPath(cluster, backup string) string {
 	return fmt.Sprintf("projects/%s/instances/%s/clusters/%s/backups/%s", ac.project, ac.instance, cluster, backup)
 }
 
+// EncryptionInfo represents the encryption info of a table.
+type EncryptionInfo struct {
+	EncryptionStatus *Status
+	EncryptionType   EncryptionType
+	KMSKeyVersion    string
+}
+
+func newEncryptionInfo(pbInfo *btapb.EncryptionInfo) *EncryptionInfo {
+	if pbInfo == nil {
+		return nil
+	}
+	info := EncryptionInfo{}
```

Comment:
nit: this could just be

```
return &EncryptionInfo{
  EncryptionStatus: pbInfo.EncryptionStatus,
  EncryptionType: EncryptionType(pbInfo.EncryptionType.Number()),
  KMSKeyVersion: pbInfo.KmsKeyVersion,
}
```